jlm/rvsdg/region.cpp

```diff
@@ -515,6 +515,39 @@ Region::ToString(const util::Annotation & annotation, char labelValueSeparator)
   return util::strfmt(annotation.Label(), labelValueSeparator, value);
 }
 
+static size_t
+computeDepth(const Node & node, std::unordered_map<const Node *, size_t> & depthMap)
```

Comment:
is it really that you want to recursively compute depth? why not TopDownTraverser?

Reply:
Used top-down traverser